ext/json/json_parser.y

```diff
@@ -270,12 +271,17 @@ int php_json_parser_object_update(php_json_parser *parser, zval *object, zend_st
 		ZVAL_NEW_STR(&zkey, key);
 		zend_std_write_property(object, &zkey, zvalue, NULL); 
 
-		if (Z_REFCOUNTED_P(zvalue)) {
-			Z_DELREF_P(zvalue);
-		}
+		zval_dtor(zvalue);
```

Comment:
The value is dtored later so we don't need to call ti now. The only thing that we need to do is `Z_DELREF_P(zvalue)` because there is `Z_ADDREF_P`. Technically zvalue will be always refcounted so we could just do `Z_DELREF_P(zvalue)` but zval_dtor is unnecessary.